great_expectations/self_check/util.py

```diff
@@ -1388,7 +1389,7 @@ def get_test_validator_with_data(
 def build_pandas_validator_with_data(
     df: pd.DataFrame,
     batch_definition: Optional[BatchDefinition] = None,
-    context: Optional["DataContext"] = None,  # noqa: F821
+    context: Optional["DataContext"] = None,
```

Comment:
@Kilo59 Having `  # noqa: F821` suppresses the PyCharm warning -- are we sure we want to remove it?

Reply:
I've fixed the error.
`DataContext` is now imported under a `if TYPE_CHECKING` block.

Reply:
❤️